src/compiler/program.ts

```diff
@@ -1983,7 +1983,7 @@ namespace ts {
                     }

```

Comment:
what?

Reply:
in the title for program.ts, it says that program.ts has 14 lines changed and also changed permissions from 100755 to 100644. I think that is safer since I don't think we can run program.ts from the command line, but I wasn't sure what prompted the change.